cesium/db.go

```diff
@@ -67,7 +67,7 @@ func (db *DB) WriteArray(ctx context.Context, key core.ChannelKey, start telem.T
 }
 
 // Read implements DB.
-func (db *DB) Read(ctx context.Context, tr telem.TimeRange, keys ...core.ChannelKey) (frame Frame, err error) {
+func (db *DB) Read(_ context.Context, tr telem.TimeRange, keys ...core.ChannelKey) (frame Frame, err error) {
```

Comment:
is context used in this function? Otherwise why have it even as optional?

Reply:
This implements an interface for forward compatibility